nvidia-gpu/vars.yaml

```diff
@@ -7,8 +7,10 @@ NVIDIA_DRIVER_PRODUCTION_VERSION: 550.144.03
 CONTAINER_TOOLKIT_VERSION: v1.17.3
```

Comment:
we need to bump this too, both always gets bumped in unison, and I think nvidia won't work when there's a version mismatch

Reply:
Whoops, right, thanks, fixing